packages/docs/docs/docs/erc20_onboarding.md

```diff
@@ -220,8 +220,8 @@ or open a new one against the network where your legacy token is deployed. Then,
 commands of the previous step:
 
 ```console
-truffle(local)> erc20Migrator = ERC20Migrator.at('ERC20_MIGRATOR_ADDRESS')
-truffle(local)> upgradeableToken = MyUpgradeableToken.at('UPGRADEABLE_TOKEN_ADDRESS')
+truffle(local)> ERC20Migrator.at('ERC20_MIGRATOR_ADDRESS').then(i => erc20Migrator = i)
```

Comment:
I know this will do the trick, but isn't it a bit dirty? I mean, defining a global variable inside the arrow function could be a bit confusing imo.
Is there any other way of accomplishing this, working on a node console?

Reply:
It's the simpler version that came to my mind, are you thinking about any other? Using a plain node console would require more setup making these examples longer. What we can discuss that afterward if we think it's better for the doc examples

Reply:
> are you thinking about any other?
Not really :disappointed:. Handling promises inside the console is annoying.